src/test/java/com/fontys/api/service/TournamentServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package com.fontys.api.service;
+
+import com.fontys.api.entities.Tournament;
+import com.fontys.api.entities.User;
+import com.fontys.api.mockrepositories.MockTournamentRepository;
+import com.fontys.api.mockrepositories.MockUserRepository;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.naming.directory.InvalidAttributeValueException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class TournamentServiceTest
+{
+    private MockTournamentRepository tournamentRepository;
+    private MockUserRepository userRepository;
+    private TournamentService tournamentService;
+
+    @Before
+    public void setUp()
+    {
+        tournamentRepository = new MockTournamentRepository();
+        userRepository = new MockUserRepository();
+        tournamentService = new TournamentService(tournamentRepository, userRepository);
+    }
+
+    @Test
+    public void createTournamentTests()
```

Comment:
I think we want to split these tests, so we can see which specific tests fails (if any) instead of only knowing that some of these tests failed (easier debugging).